hathor/transaction/transaction.py

```diff
@@ -128,10 +128,10 @@ def create_from_proto(cls, tx_proto: protos.BaseTransaction,
         return tx
 
     @classmethod
-    def create_from_struct(cls, struct_bytes: bytes,
-                           storage: Optional['TransactionStorage'] = None) -> 'Transaction':
+    def create_from_struct(cls, struct_bytes: bytes, storage: Optional['TransactionStorage'] = None,
+                           *, verbose: bool = False) -> 'Transaction':
```

Comment:
Should we also log `nonce` here when `verbose=True`?

Reply:
Fixed. Thanks!